docset/windows/hyper-v/Resize-VHD.md

```diff
@@ -38,15 +38,13 @@ Resize-VHD [-Path] <String[]> [-ToMinimumSize] [-AsJob] [-Passthru] [-CimSession
 ```
 
 ## DESCRIPTION
-The **Resize-VHD** cmdlet resizes a virtual hard disk.
-This cmdlet lets you shrink or expand the size of a virtual hard disk, but the shrink operation is allowed only on VHDX virtual hard disks.
+The **Resize-VHD** cmdlet changes the maximum physical size of a virtual hard disk. It can expand both VHD and VHDX files. It can shrink only VHDX.
```

Comment:
```suggestion
The **Resize-VHD** cmdlet changes the maximum physical size of a virtual hard disk. It can expand both VHD and VHDX files but can shrink only VHDX files.
```